vispy/scene/cameras/cameras.py

```diff
@@ -1005,6 +1005,8 @@ def viewbox_mouse_event(self, event):
                 norm = np.mean(self._viewbox.size)
                 if self._event_value is None:
                     self._event_value = self.center
+                elif len(self._event_value) == 2:
+                    self._event_value = self.center
```

Comment:
a nitpick on a fix of this few lines, but it is more DRY to do:
```
if self._event_value is None or len(self._event_value) == 2:
```
that way the operation is only called once